### PR TITLE
User res

### LINKED ISSUE #28: Create a my reservations page for the user view

### LINKED ISSUE #33: Connect the my reservations page to the user home

### LINKED ISSUE #42: Change color scheme of user home to be razorback colors

### PR DESCRIPTION
**Pull request by ceborn**
Made user home and my reservations razorback theme. Connected user home to my reservations page. Made a static my reservations page for the user view.

### CODE REVIEW

**Comment by ceborn** [comment]
Connected user home to user reservations page. Made user home and user res pages razorback theme. Created a user my reservations page.

File: src/components/User/UserRes.vue [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        >
+        <v-toolbar-title
+          class="text-h5"
+        >
+          My Reservations
+        </v-toolbar-title>
+        <v-btn
+          depressed
+          :right="true"
+          :absolute="true"
```
Comment (greyhixson) [inline src/components/User/UserRes.vue]: `:right="true"` is the same thing as just writing `right`.  Change them both to `right` and `absolute` to make it more readable. 

```suggestion
          right
          absolute
```

File: src/components/User/UserHome.vue [hunk 2]
```diff
@@ -1,37 +1,59 @@
...
+
+        <v-toolbar-title
+          class="text-h5"
+        >
+          View Booking Portal
+        </v-toolbar-title>
+        <v-btn
+          depressed
+          :right="true"
+          :absolute="true"
```
Comment (greyhixson) [inline src/components/User/UserHome.vue]: Similar comment left in UserRes
```suggestion
          right
          absolute
```

**Review by greyhixson — CHANGES_REQUESTED** [review]
Looks good, just a couple minor things that need to be fixed

File: src/views/UserResView.vue [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import UserRes from '../components/User/UserRes.vue';
+
+export default {
+  name: 'UserView',
+
+  components: {
+    UserRes,
+  },
+};
+</script>
```
Comment (greyhixson) [inline src/views/UserResView.vue]: File shouldn't exist. Instructions on why left in review comment.

File: src/router/index.js [hunk 4]
```diff
@@ -21,6 +22,11 @@ const routes = [
     name: 'Profile',
     // add component later
   },
+  {
+    path: '/UserResView',
+    name: 'UserResView',
+    component: UserResView,
+  },
```
Comment (greyhixson) [inline src/router/index.js]: So close! Instead, just import your UserRes component and change the path to /UserRes. UserResView is unnecessary. 
```suggestion
  {
    path: '/UserRes',
    name: 'UserRes',
    component: UserRes,
  },
```

File: src/router/index.js [hunk 5]
```diff
@@ -2,6 +2,7 @@ import Vue from 'vue';
 import VueRouter from 'vue-router';
 import AdminView from '../views/AdminView.vue';
 import UserView from '../views/UserView.vue';
+import UserResView from '../views/UserResView.vue';
```
Comment (greyhixson) [inline src/router/index.js]:
```suggestion
import UserRes from '../components/User/UserRes.vue';
```

File: src/components/User/UserHome.vue [hunk 6]
```diff
@@ -1,37 +1,59 @@
...
+      <v-banner
+        single-line
+        height="75%"
+      />
+    </v-card>
     <v-btn
       color="#E0E0E0"
-      class="mb-4"
+      class="mb-8"
+      href="/#/UserResView"
```
Comment (greyhixson) [inline src/components/User/UserHome.vue]:
```suggestion
      href="/#/UserRes"
```

**Review by greyhixson — CHANGES_REQUESTED** [review]
Looks good, a couple of minor changes and one larger one. You got super close to getting the router set up right, but UserResView is unnecessary. I added more info in the comments.

File: src/components/User/UserRes.vue (hunk 1 above)
Comment (ceborn) [reply src/components/User/UserRes.vue]: Done

File: src/components/User/UserHome.vue (hunk 2 above)
Comment (ceborn) [reply src/components/User/UserHome.vue]: Done

File: src/router/index.js (hunk 4 above)
Comment (ceborn) [reply src/router/index.js]: Done

File: src/router/index.js (hunk 5 above)
Comment (ceborn) [reply src/router/index.js]: Done

File: src/components/User/UserHome.vue (hunk 6 above)
Comment (ceborn) [reply src/components/User/UserHome.vue]: Done

**Comment by ceborn** [comment]
I fixed the requested changes. Could you review this when you get the chance?

**Review by greyhixson — APPROVED** [review]
Great job!

**Pull request merged**